content/docs/integrations/external-dns-cert-manager.md

```diff
@@ -1,21 +1,28 @@
 ---
-title: ExternalDNS & Cert Manager
+title: ExternalDNS and cert-manager
 weight: 520
-description: Use kgateway with External DNS and Cert Manager. 
 ---
 
-[ExternalDNS](https://github.com/kubernetes-sigs/external-dns) and [Cert Manager](https://github.com/cert-manager/cert-manager) are two well known integrations within the Kubernetes ecosystem that can be used in conjunction to automate the creation of TLS certificates. 
+[ExternalDNS](https://github.com/kubernetes-sigs/external-dns) and [cert-manager](https://github.com/cert-manager/cert-manager) are two well-known integrations within the Kubernetes ecosystem that can be used in conjunction to automate the creation of TLS certificates. 
+
+* **ExternalDNS**: Instead of manually editing your domain in your DNS provider to add load balancer IP addresses, you can use ExternalDNS to dynamically set up and control DNS records for discovered gateway and HTTP resources. When you define a hostname in an HTTPRoute resource, ExternalDNS uses the external address that is assigned to the gateway's load balancer service that serves this hostname, and uses this information to create a DNS record in the DNS provider that you configured.
```

Comment:
```suggestion
* **ExternalDNS**: Instead of manually editing your domain in your DNS provider to add load balancer IP addresses, you can use ExternalDNS to dynamically set up and control DNS records for discovered Gateway and HTTPRoute resources. When you define a hostname in an HTTPRoute resource, ExternalDNS uses the external address that is assigned to the Gateway's load balancer service that serves this hostname, and uses this information to create a DNS record in the DNS provider that you configured.
```